lib/tracker_api/endpoints/search.rb

```diff
@@ -0,0 +1,53 @@
+module TrackerApi
+  module Endpoints
+    class Search
+      attr_accessor :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def get(project_id, query, params={})
+        raise ArgumentError, 'Valid query required to search' unless query.is_a?(String)
+
+        params.key?(:body) ? params[:body][:query] = query : params[:body] = { query: query }
+        data = client.paginate("/projects/#{project_id}/search", params)
+
+        raise Errors::UnexpectedData, 'Hash of search results expect' unless data.is_a? Hash
+
+        create_search_results(data)
```

Comment:
I think Virtus does all the conversion for you so you might be able to return `Resources::SearchResultContainer.new(data)` here.